app/app.py

```diff
@@ -173,22 +173,32 @@ def _handle_premis_event(event: PremisEvent, mh_client: MediaHaven):
 
         s3_bucket = fragment_info["s3_bucket"]
         s3_object_key = fragment_info["s3_object_key"]
+        # If we have a collateral (subtitle): no need for een archivedEvent
+        if s3_bucket == 'mam-collaterals':
+            log.info(
```

Comment:
Open vraag, is info het meest gepaste log level?

Reply:
Discutabel, idd. We zitten al in een correct event (de gedropte zijn debug) en van een collateral, die sowieso minder voorkomen. _I'll leave it for now._